osx-manual-setup.md

```diff
@@ -26,8 +26,7 @@ ln -s .tool-versions.sample .tool-versions
 
 ### Ruby and Node.js
 
-The project supports **[ruby 2.6.x](https://www.ruby-lang.org/en/downloads/)** and **[Node.js 10](https://nodejs.org/en/download/)**.
-The recommended way to install them is with `asdf`:
+The project supports **[ruby 2.6.x](https://www.ruby-lang.org/en/downloads/)** and **[Node.js 10](https://nodejs.org/en/download/)**. To install them with `asdf` run:
```

Comment:
node 12, no?

Reply:
if you have ready node 14 support, I'm ok if you directly recommend 14 or later. 12 is gone from fedora anyway.

Reply:
I see this is macos but still version question applies.

Reply:
Let's go with 12 for now